docs/content/operations/dashboard.md

```diff
@@ -51,12 +59,39 @@ api:
 --api.dashboard=true
 ```
 
-!!! important "API/Dashboard Security" 
-    
-    To secure your dashboard, the use of a `service` named `api@internal` is mandatory and requires the definition of a router using one or more security [middlewares](../middlewares/overview.md)
-    like authentication ([basicAuth](../middlewares/basicauth.md) , [digestAuth](../middlewares/digestauth.md), [forwardAuth](../middlewares/forwardauth.md)) or [whitelisting](../middlewares/ipwhitelist.md).  
-    More information about `api@internal` can be found in the [API documentation](./api.md#configuration)
+Then specifies a router associated to the service `api@internal` to allow:
+
+- Defining one or more security features through [middlewares](../middlewares/overview.md)
+  like authentication ([basicAuth](../middlewares/basicauth.md) , [digestAuth](../middlewares/digestauth.md),
+  [forwardAuth](../middlewares/forwardauth.md)) or [whitelisting](../middlewares/ipwhitelist.md).
+
+- Defining your own [HTTP routing rule](../../routing/routers/#rule) for accessing the dashboard,
+  through Traefik itself (sometimes referred as "Traefik-ception").
+
+Please visit the ["Configuration" section of the API documentation](./api.md#configuration)
+to learn about configuring a router with the service `api@internal` and enabling the security features.
+
+## Insecured Mode
+
+This mode is not recommend because it does not allow usage of security features.
+
+To enable the "insecure mode", use the following options from [Traefik's API](./api.md#insecure):
```

Comment:
you had actually gotten it right here ;P

Reply:
:trollface: